mewarpx/mewarpx/util.py

```diff
@@ -4,15 +4,13 @@
 import errno
 import inspect
 import os
-
 from pywarpx import geometry
 from mewarpx import mwxconstants as constants
 
 # http://stackoverflow.com/questions/50499/in-python-how-do-i-get-the-path-and-name-of-the-file-t
 mewarpx_dir = os.path.dirname(os.path.abspath(
     inspect.getfile(inspect.currentframe())))
 
-
```

Comment:
I believe it's standard to have 2 blank lines between non-class functions, this includes between the header imports and the first function definition.